src/components/sendWritable/index.test.js

```diff
@@ -23,10 +25,21 @@ describe('SendWritableHOC', () => {
     });
     store.subscribe = () => {};
     store.dispatch = () => {};
-    wrapper = mount(<Router><Provider store={store}><SendHOC i18n={i18n} /></Provider></Router>);
+    wrapper = mount(<Router>
+      <Provider store={store}>
+        <SendHOC i18n={i18n} settings={{}} />
+      </Provider>
+    </Router>);
   });
 
   it('should render Send', () => {
     expect(wrapper.find('SendWritable')).to.have.lengthOf(1);
   });
+
+  it('should bind settingsUpdated action to SendWritable props.settingsUpdated', () => {
+    const actionsSpy = spy(settingsActions, 'settingsUpdated');
+    wrapper.find('SendWritable').props().settingsUpdated({});
```

Comment:
what about passing a valid settings object and asserting that the call contains the key of the object?